src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -449,7 +479,11 @@ def _build_for_jumpstart(self):
         self.secret_key = None
         self.jumpstart = True
 
-        pysdk_model = self._create_pre_trained_js_model()
+        pysdk_model = (
+            self.pysdk_model
+            if self.pysdk_model is not None
```

Comment:
You need to use getattr instead, otherwise this will throw no attribute exception. ie. `getattr(self, "pysdk_model", self._create_pre_trained_js_model())`